tests/component_handlers/kafka_connect/test_connect_handler.py

```diff
@@ -53,17 +53,18 @@ def connector_handler(connect_wrapper: MagicMock) -> KafkaConnectHandler:
             timeout=0,
         )
 
-    def test_should_create_connector_in_dry_run(
+    @pytest.mark.asyncio
+    async def test_should_create_connector_in_dry_run(
         self,
         renderer_diff_mock: MagicMock,
         log_info_mock: MagicMock,
     ):
```

Comment:
Small but nice to have. Instead of repeating yourself, you can create this fixture. 
```python
    @pytest.fixture
    def connector_wrapper(self) -> MagicMock:
        return AsyncMock()
```

Then you can pass it to the test method parameter:

```suggestion
    @pytest.mark.asyncio
    async def test_should_create_connector_in_dry_run(
        self,
        renderer_diff_mock: MagicMock,
        log_info_mock: MagicMock,
        connector_wrapper: MagicMock
    ):
```